reinforcement_learning/bindings/cs/rl.net.cli/rl.net.cli.csproj

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Is it possible for this to be sdk based now? Given that's where we are moving everything else

Reply:
Yes, I just had a bit of a hard time generating an exe out of the SDK-style project, because I forgot I had to explicitly set net462 as my target framework, rather than NetCoreApp.